### DISCUSSION TITLE
Flex Not Working...Any Ideas?

### DISCUSSION THREAD

**Discussion by danielbirch** [tailwindlabs/tailwindcss#10963] [Help] [answered]
Hey everyone,

I'm new to Tailwind, however not new to CSS. I've just started putting a layout together and the flex property is not working as expected. The expected behaviour is for the div's to line up in a row, however they are not. This is the code:

```
<div class="flex flex-row w-screen">
    <div class="flex-auto w-1/4">Logo</div>
    <div class="flex-auto w-2/4">
      <nav>
        <a href="#">About Me</a>
        <a href="#">Skills</a>
        <a href="#">Explore Portfolio</a>
        <a href="#">Resources</a>
      </nav>
    </div>
    <div class="flex-auto w-1/4">Contact Btn</div>
  </div>
```

And this is the output:

<img width="1470" alt="Screenshot 2023-04-06 at 7 15 18 PM" src="https://user-images.githubusercontent.com/122473191/230437534-14966514-c662-426e-8d96-114f3f3971da.png">

Any help would be greatly appreciated, I'm not sure what I'm missing. Tailwind is integrated properly, other utility classes are recognised, but flex and flex-row don't seem to be doing anything.

**Comment by wongjn** [comment]
Despite your assertion, it seems like Tailwind is not integrated properly for this piece of DOM. Consider checking that the `content` file globs in the Tailwind configuration sufficiently cover the file that the code resides in. Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by danielbirch** [reply, marked as answer]
Thanks for the prompt response. Yeah I actually just discovered that I didn't have the CLI build process running. 🤪

**Reply by keshav-15151** [reply]
The same error occurs for me too, flex isn't working on the tailwind css code. Any suggestions?

**Reply by Dubey145** [reply]
@keshav-15151 you need to check the contents in tailwind.config.ts. The file in which you are using tailwind should be included in the contents section
![Screenshot 2025-01-06 at 11 14 25 AM](https://github.com/user-attachments/assets/8e4a76c6-60d1-4d89-8b4a-732fe4fa7acc)
